app/components/braveShields/braveShieldsStats.tsx

```diff
@@ -75,48 +125,59 @@ export default class BraveShieldsStats extends React.Component<BraveShieldsStats
       >
         <Column theme={theme.statsNumbers} size={1}>
           <TextLabel
+            id='totalAdsTrackersBlockedStat'
             theme={theme.totalAdsTrackersBlockedStat}
             text={this.totalAdsTrackersBlocked}
-            onClick={this.onClickadsTrackersBlockedStats}
+            onClick={this.onToggleBlockedResourcesDetails}
```

Comment:
In a follow up, please make this use a hand cursor (only for ads blocked stat) and make it toggle if you click twice.